### PR TITLE
[BUGFIX] separable convolution

### PR DESCRIPTION
**Pull request by ashutoshparkhi**
Number of groups were not being calculated correctly for (NHWC,  HWIO) pair

Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 1]
```diff
@@ -1047,6 +1047,48 @@ def test_forward_rel_ops():
...
+                      dw_filter,
+                      pw_filter,
+                      strides=strides,
+                      padding=padding,
+                      data_format=data_format)
+
+        compare_tf_with_tvm(np.reshape(data_array, tensor_in_sizes).astype('float32'),
+                            'Placeholder:0', 'separable_conv2d:0')
+
+def test_forward_sep_convolution():
```
Comment (vinx13) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: run this test in main

File: nnvm/src/top/nn/convolution.cc [hunk 2]
```diff
@@ -68,8 +68,12 @@ inline bool Conv2DInferShape(const nnvm::NodeAttrs& attrs,
       << "incorrect stride size: " << param.strides;
   CHECK_EQ(param.dilation.ndim(), 2U)
       << "incorrect dilate size: " << param.dilation;
-  CHECK_EQ(dshape[1] % param.groups, 0U)
-      << "input channels must divide group size";
+
+  // In case of depthwise, groups is set to (depth_mult * channels)
+  if (param.groups == 1)
```
Comment (vinx13) [inline nnvm/src/top/nn/convolution.cc]: `param.groups` != 1 ?

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 3]
```diff
@@ -1047,6 +1047,48 @@ def test_forward_rel_ops():
...
+    _test_sep_convolution([1, 32, 32, 3], [7, 7, 3, 8], [1, 1, 24, 64], [1, 1], [1, 1], 'SAME', 'NHWC')
```
Comment (srkreddy1238) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Pls add another case for NCHW.

Comment (srkreddy1238) [reply nnvm/tests/python/frontend/tensorflow/test_forward.py]: you may refer tensorflow test cases for params which cover most of the models.

Comment (JaydeepIMG) [reply nnvm/tests/python/frontend/tensorflow/test_forward.py]: Working on it.

**Comment by yzhliu** [comment]
could you please update the PR according to the reviews? @ashutoshparkhi

**Comment by tqchen** [comment]
close due to inactive status

**Pull request closed without merging**